rasa/core/events/__init__.py

```diff
@@ -308,6 +318,13 @@ def as_dict(self) -> Dict[Text, Any]:
         )
         return _dict
 
+    def as_dict_core(self) -> Dict[Text, Union[None, Text, List[Optional[Text]]]]:
```

Comment:
but it contains also a lot of unneeded information in the different format that is harder to use in policies core featurizers

Reply:
i think this is optional.


imo though this doesn't look like a function that is used outside of a featurizer so it might more sense to add it there

Reply:
it is used inside domain, and is different for different events

Reply:
I agree that it is optional, but in my opinion, having it inside corresponding events nicely split responsibility, because it's events responsibility to tell domain how it wants to be featurized